src/konfo_indeksoija_service/kouta/toteutus.clj

```diff
@@ -0,0 +1,10 @@
+(ns konfo-indeksoija-service.kouta.toteutus
+  (:require [konfo-indeksoija-service.rest.kouta :as kouta-backend]
+            [konfo-indeksoija-service.rest.koodisto :refer [get-koodi-nimi-with-cache]]
+            [konfo-indeksoija-service.kouta.common :as common]))
+
+(defn create-index-entry
+  [oid]
+  (let [toteutus (common/complete-entry (kouta-backend/get-toteutus oid))
```

Comment:
Kun tehdään näitä verkkokutsuja niin oisko kannattavaa käyttää futureja jolloin molemmat kutsut tapahtuis samaan aikaan?

Reply:
Laitetaan tämä TODO-listalle tehtäväksi siinä vaiheessa, jos tulee tarvetta optimoida indeksojaa.

Reply:
https://jira.oph.ware.fi/jira/browse/KTO-103